src/index.html

```diff
@@ -17,11 +17,381 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="style.scss"
+    />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
     />
   </head>
 
+  <header class="header">
+    <a
+      href="#"
+      class="header__logo"
+    >
+      <img
+        src="/src/images/logo.png"
+        alt="logo"
+        class="header__logo-img"
+      />
+    </a>
+
+    <nav class="header__nav">
+      <ul class="header__nav-list">
+        <li class="header__nav-list-item">
+          <a
+            href="#"
+            class="header__nav-list-item-link is-active"
+          >
+            Apple
+          </a>
+        </li>
+
+        <li class="header__nav-list-item">
+          <a
+            href="#"
+            class="header__nav-list-item-link"
+          >
+            Samsung
+          </a>
+        </li>
+
+        <li class="header__nav-list-item">
+          <a
+            href="#"
+            class="header__nav-list-item-link"
+          >
+            Smartphones
+          </a>
+        </li>
+
+        <li class="header__nav-list-item">
+          <a
+            href="#"
+            data-qa="nav-hover"
+            class="header__nav-list-item-link"
+          >
+            Laptops & Computers
+          </a>
+        </li>
+
+        <li class="header__nav-list-item">
+          <a
+            href="#"
+            class="header__nav-list-item-link"
+          >
+            Gadgets
+          </a>
+        </li>
+
+        <li class="header__nav-list-item">
+          <a
+            href="#"
+            class="header__nav-list-item-link"
+          >
+            Tablets
+          </a>
+        </li>
+
+        <li class="header__nav-list-item">
+          <a
+            href="#"
+            class="header__nav-list-item-link"
+          >
+            Photo
+          </a>
+        </li>
+
+        <li class="header__nav-list-item">
+          <a
+            href="#"
+            class="header__nav-list-item-link"
+          >
+            Video
+          </a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+
   <body>
```

Comment:
The body tag should be outside and wrap the header and main tags. This is because the body element represents the content of the HTML document and the header and main are part of this content.